engine/jsonlint/pipe.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://repometric.github.io/linterhub/schema/extr.json",
+    "id": "jsonlint",
+    "name": "jsonlint",
+    "pipeline": [
+        {
+            "cmd": "jsonlint",
+            "success": 1,
+            "engine": true,
+            "args":{
+                "--compact":"",
+				"2>&1":""
```

Comment:
I suggest to add an extra property to pipeline item: stream: (stdout|stderr) with default value stdout - it will specify what to pass to the next pipeline item (stdout or stderr of the previous pipe). Then this redirect (2&1) will not be needed and should be handled in source code.

Reply:
sounds good

Reply:
Create issue in `ep` by number [47](../../ep/issues/47)